modules/localgov_campaigns_paragraphs/js/localgov-accordion.js

```diff
@@ -32,63 +32,111 @@
+                var openPaneButton = accordion
+                  .querySelectorAll('[aria-controls="' + openPaneId + '"]');
+                openPane[0].classList.remove(openClass);
+                openPaneButton[0].setAttribute('aria-expanded', 'false');
+              }
+
+              // Show new pane.
+              e.target.setAttribute('aria-expanded', 'true');
+              targetPane[0].classList.add(openClass);
+            }
+            else {
+              // If target pane is currently open, close it.
+              e.target.setAttribute('aria-expanded', 'false');
+              targetPane[0].classList.remove(openClass);
             }
+          });
+
+          // Add show/hide button to each accordion pane title element.
+          title[0].children[0].innerHTML = '';
+          title[0].children[0].appendChild(button);
+        }
+
+        // Add init class.
+        accordion.classList.add(initClass);
+
+      };
 
-            // Show new pane.
-            e.target.setAttribute('aria-expanded', 'true');
-            targetPane[0].classList.add(openClass);
+      var destroy = function destroy() {
+        for (var i = 0; i < numberOfPanes; i++) {
+          // Remove buttons from accordion pane titles.
+          var button = accordion.querySelectorAll('.accordion-pane__title')[i]
+            .querySelectorAll('button');
+          if (button.length > 0) {
+            button[0].outerHTML = button[0].innerHTML;
           }
-          else {
-            // If target pane is currently open, close it.
-            e.target.setAttribute('aria-expanded', 'false');
-            targetPane[0].classList.remove(openClass);
+
+          // Remove id attributes from pane content elements.
+          accordionPanes[i].querySelectorAll('.accordion-pane__content')[0]
+            .removeAttribute('id');
+
+          // Remove open class from accordion pane's content elements.
+          if (accordionPanes[i].querySelectorAll('.accordion-pane__content')[0]
+            .classList.contains(openClass))
+          {
+            accordionPanes[i].querySelectorAll('.accordion-pane__content')[0]
+            .classList.remove(openClass);
           }
-        });
+        }
+        // Remove accordion init class.
+        accordion.classList.remove(initClass);
+      };
 
-        // Add show/hide button to each accordion pane title element.
-        title[0].children[0].innerHTML = '';
-        title[0].children[0].appendChild(button);
+      var breakpointCheck = function breakpointCheck() {
+        if (mq.matches || breakpoint === null) {
+          create();
+        } else {
+          destroy();
+        }
+      };
 
-        // Add init class.
-        accordion.classList.add(initClass);
+      // Trigger create/destroy functions at different screen widths
+      // based on the value of data-accordion-tabs-switch attribute.
+      if (window.matchMedia) {
+        var mq = window.matchMedia(`(max-width: ${breakpoint})`);
```

Comment:
ES6 Templates literals [don't work in IE11](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Browser_compatibility).  We, at least in Croydon, have to support IE11.

I can see two options here:
- Stop supporting IE11 :)
- Follow the [lead of Drupal core](https://www.drupal.org/node/2815083) and provide two copies of this file - localgov-accordion.es6.js and localgov-accordion.js.  The later produced by Babel.

@gosia-mlynarczyk @finnlewis @cjstevens78

Reply:
I vote for option 1! 😉 

Ok, I've added source files (.es6.js) for both the accordion and tabs.

Reply:
+10 for adding the ES6 source file.